src/java/org/apache/cassandra/io/sstable/format/SSTableScanner.java

```diff
@@ -19,7 +19,6 @@
 
 import java.io.IOException;
 import java.util.ArrayList;
-import java.util.Collection;
```

Comment:
Reverted this code, and the code in BigTableScanner to the version on trunk as we no longer need these changes.